### ISSUE TITLE
[编译问题] 编译问题描述(必填)

### ISSUE THREAD

**Issue by CoderAbeliu** [ZLMediaKit/ZLMediaKit#3729]
## 相关日志及环境信息

**iOS参考文档生成xcode 可用工程后，编译出对应的静态库，但并未提供头文件用以调用**
![3EDC72BB-4803-4B8A-8D46-9F1D9A17F324](https://github.com/user-attachments/assets/be1f5934-71a1-4a0e-8da6-8f3077f968c0)
![735600AD-0D78-4469-BA74-D433E9779396](https://github.com/user-attachments/assets/968698ed-2ccb-4183-89a0-13dec3423c1f)

<details>
<summary>展开查看详细编译日志</summary>
<pre>

```
详细日志粘在这里!
```

</pre>
</details>

[CMakeCache.txt](https://github.com/user-attachments/files/16279158/CMakeCache.txt)

## 各种环境信息

Xcode 15.4 
* **代码提交记录/git commit hash**:
* **操作系统及版本**:
* **硬件信息**:
* **其他需要补充的信息**:

**Comment by xia-chu** [comment]
是的 没install头文件 ios工程其实也没官方直接支持。建议用存c的mk_api 这个头文件比较少，如果你要用c++的接口 那么要自己整理下所有头文件
